OWML.Launcher/Program.cs

```diff
@@ -13,32 +13,44 @@ public class Program
     {
         static void Main(string[] args)
         {
-            var owmlConfig = GetOwmlConfig();
+            var owmlConfig = GetOwmlConfig() ?? CreateOwmlConfig();
```

Comment:
Won't this cause problems if you add new entries to the config? As in, if a user has a OWML.Config.json file, and then in a new update OWML adds a new configurable setting. The config exists, but is missing a property, so you're not getting that property from the default config.

Reply:
I tried deleting a line from the config, to simulate your situation. It worked fine :-)
The config defaulted to the default of the data type (false for bool in this case) - it showed fine in the menu and saved properly after changing it.

Reply:
Hm but what if default was supposed to be true? 😅

Reply:
Hmmm... 🤔 
I don't know how to code what you're suggesting without making a manually maintained mess. Any ideas?

Reply:
I guess iterating over properties using reflection. Sounds hard :P

Reply:
I guess too many things would have to change. What makes most sense for me is not to copy properties selectively, but to do in the moment of getting the property. So when you're trying to get OwmlConfig.GamePath, the getter checks if the property exists in OWML.Config.json, and if not, gets it from OWML.DefaultConfig.json.

Ideally, you wouldn't even need to create the OWML.Config.json file until the user changes a setting. And when they do, the user config would only need to include the settings that differ from the default.

But I understand that this can be to big of a change, and might not be easy to do with the way the JSON serialization is done (I think it always writes the whole file from the properties).

In any case, it's at the very least something to keep in mind if we add new settings ;)